src/app/newSettings/Sections/Account/Plans/utils/planUtils.ts

```diff
@@ -46,26 +48,30 @@ const getCurrentChangePlanType = ({
   return 'free';
 };
 
-const getPlanName = (storagePlan: StoragePlan | null) => {
-  return storagePlan?.simpleName ?? FreeStoragePlan.simpleName;
+const getPlanName = (storagePlan: StoragePlan | null, limit?: number) => {
+  if (storagePlan?.simpleName) return storagePlan?.simpleName;
+  if (limit) return bytesToString(limit, false);
+  return FreeStoragePlan.simpleName;
 };
-const getCurrentUsage = (plan: PlanState) => {
-  return plan.usageDetails?.total ?? -1;
+const getCurrentUsage = (usage: UsageResponse | null) => {
+  return usage?.total ?? -1;
 };
 
 const getPlanInfo = (storagePlan: StoragePlan | null) => {
   if (storagePlan) {
     if (storagePlan.paymentInterval === RenewalPeriod.Annually) {
+      const priceTruncated = Math.trunc(storagePlan.price * 100) / 100;
       return (
+        `${priceTruncated} ` +
         moneyService.getCurrencySymbol(storagePlan.currency) +
```

Comment:
Are these changes in the position of the currency symbol due to the design? If not, leave it as it is, as it will be there for a reason. I have seen in several files this change, that's why I ask.

Reply:
Of course, I used this design as a base, so I make the necessary changes to achieve the same consistency in the sections.
https://www.figma.com/design/62hpKnJYlxUN77HUECXAlF/Drive-Web?node-id=10128-15971